velox/docs/README.md

```diff
@@ -45,3 +45,34 @@ git commit -m "Update documentation"
 # This will get the website updated.
 git push -f upstream update-docs:gh-pages
 ```
+
+## Monthly Updates
+The monthly update report is generated manually using the following commands and template.
+
+Command to get the list of contributors and their contribution count in a specific month.
+```
+git log --since='June 1 2024' --until='June 30 2024' --pretty=format:"%an" | sort | uniq -c
+```
+Command to get the list of contributions in a specific month.
+```
+git log --since='June 1 2024' --until='June 30 2024' --pretty=format:"%h%x09%an%x09%ad%x09%s"
+```
+
+The following template is used to write a monthly update report. The main sections
+are as follows.
+- Documentation
+- Core Library
+- Presto Functions
+- Spark Functions
+- Hive Connector
+- Performance and Correctness
+- Build System
+- Credits
+
+Below are a few monthly update guidelines.
+- Each update should be in the imperative, present tense similar to a git commit.
+- In sections where applicable, start with the new additions with `Add` followed by
```

Comment:
nit: Maybe also specify to, Start with 'Adds' followed by Fixes , followed by other .